README.md

```diff
@@ -13,6 +13,7 @@ To only compile the application manually you'll need to install:
 - [sqlx-cli](https://crates.io/crates/sqlx-cli)
 - [sea-orm-cli](https://www.sea-ql.org/SeaORM/docs/generate-entity/sea-orm-cli/)
 - [openapi-generator](https://openapi-generator.tech/docs/installation/)
+- [gsed](https://formulae.brew.sh/formula/gnu-sed)
```

Comment:
- please specify that the `gsed` is a Mac specific pre-requisite
- update the  ### Update Database Entity Files section
  - replace `make db-entity` with `cargo make db-entity`